src/features/areaAssignments/components/MapControls.tsx

```diff
@@ -14,9 +14,8 @@ const MapControls: FC<MapControlsProps> = ({ map, onFitBounds }) => {
     <ZUIMapControls
       onFitBounds={onFitBounds}
       onGeolocate={(lngLat) => {
-        map?.flyTo(flipForLeaflet(lngLat), undefined, {
+        map?.panTo(flipForLeaflet(lngLat), {
```

Comment:
I'm not sure if this is what did it, but the map has gone back to the viewport following me now. I can't pan away from my own position, because within a second or two I will be panned back.

If I understand our discussion the other day correctly, this is no longer the intended behaviour, and you intentionally removed all the follow behaviour in a previous iteration. Now that behaviour seems to have unintentionally returned.